src/wallet/rpcwallet.cpp

```diff
@@ -183,6 +185,33 @@ static std::string LabelFromValue(const UniValue& value)
     return label;
 }
 
+static void SetFeeEstimateMode(CWallet* const pwallet, CCoinControl& cc, const UniValue& estimate_mode, const UniValue& estimate_param)
```

Comment:
A bit of documentation would go a long way for function declaration.

Reply:
Added documentation!